packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -146,6 +146,8 @@ export const messages = {
     'To run this command, install the Salesforce Functions plugin. For more info, see [Getting Started with Salesforce Functions](https://github.com/forcedotcom/evergreen-docs/blob/master/Getting%20Started%20Guide/Evergreen%20Getting%20Started%20Guide.md#installing-the-salesforce-functions-sfdx-plugin).',
   force_function_start_warning_docker_not_installed_or_not_started:
     'It looks like Docker is not installed or running. To run this command, install and start Docker Desktop from [https://www.docker.com/products/docker-desktop](https://www.docker.com/products/docker-desktop)',
+  force_function_start_unexpected_error:
+    'SFDX: Start Function exited unexpectedly with code %s',
```

Comment:
@ekapner This is the message when an error code is unexpected, similar to what we have in local dev: 

https://github.com/forcedotcom/salesforcedx-vscode/blob/20266dc8760222b290f48df369bae9220a433d9a/packages/salesforcedx-vscode-lwc/src/messages/i18n.ts#L31

fyi @jag-sfdc https://github.com/forcedotcom/salesforcedx-vscode/pull/2538/commits/0f16aa0948a96ca54a0fcf2f69f5a6fcc478758c adds handling for unexpected error

Reply:
@ekapner They will have to see what the output panel says. We have handled docker installation/ plugin installation, this is a precaution to show a message where CLI starts and throw an unexpected error. For recovering from an unexpected error, generally user would need to look at the error message that CLI (Functions plugin) outputs. It would be more helpful for the user if the message is more user friendly, but there's not much we can do in VS Code.